### ISSUE TITLE
Augment "Research" page with a "by outside researchers" category

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/website#253]
@emily-phet would appreciate the following-

@zepumph on our research page http://phet.colorado.edu/en/research in the section "Publications and Presentations" lets add a subsection called "Work on PhET by outside Researchers"

Under that section will be a list of papers from researchers outside of the PhET organization.

I will forward you a set of emails from @emily-phet that include the relevant resources. There may be some duplication, so please make sure we have only unique entries.

Let me know if you have any questions...should be pretty quick and would be great to accomplish before you leave us (:cry:)

**Comment by zepumph** [comment]
A few questions. 
1) I just want to get all of the data recognized so I don't miss anything. There is the spread sheet, with 32 or something entries, the email, with 18 entries, some may be overlaps, and then on the email I got, there were two other attachments, one was the Rohmah abstract, would you like that document put up on the website in the "publications" folder? Then I got a strange file called ATT0000001.html and want to make sure that that isn't something that should be included.
2) In all other links on the research page, the title links to the paper, but on many of the items you gave me, including the entire google spreadsheet (unless I read it wrong) there are no links to the actual papers. Would you like me to try and find them, or just leave it as plain text?

Thanks

**Comment by ariel-phet** [comment]
ignore ATTblahblah.html

Links would be best if available, if no link is available leave plain text and we will get a student worker to track down links in the future.

**Comment by zepumph** [comment]
@ariel-phet  Take a look at it on phetDev and see what you think. https://phet-dev.colorado.edu/en/research
There was a lot of missing data that you may want to go out and search for on the internet. I found links for many of the ones missing them, but not all. It is a comprehensive list of pretty much all the information given to me. Tell me if it's too much, or anything should be changed. Thanks!

**Comment by emily-phet** [comment]
Here's a couple of thoughts - probably for an undergraduate student to work through:
- We'll need to make sure there's a link to this new category of papers at the top
- We'll need to make sure we're linking directly to the publisher's site for papers we can find links to - we don't want to be linking to 3rd party sites.
- If we're not going to categorize the papers by topic, it would probably be useful to have them listed in reverse chronological order (most recent pub at the top)
- For pubs in a non-English language, it would be nice to list the language in parenthesis either at the beginning or end of the reference (not sure which is better). It also might be nice to have an English translation of the title listed (I know we have this for some, if not we could see if Google translate spits out something useful)

We should also start a topical list for accessibility-related papers. I've got one published, and two which should be before the end of the year.

**Comment by ariel-phet** [comment]
@zepumph - as @emily-phet said, right under the heading where it says " Publications and Presentations" there should be a link that says **Work on PhET by outside Researchers**

@emily-phet - do you want to make a task for @elisemorgan to perhaps organize this list and follow through on the pieces you outline above?

**Comment by zepumph** [comment]
I just pushed to master a few changes:
     There is now a link at the top of the page
      I organized the list by date (the best I can because there are some that had no dates)
     I added a parenthetical at the end of the foreign translations saying the language.

See what you think, it will be on phet-dev within the hour.

**Comment by zepumph** [comment]
By that I mean it's on phet-dev now.

**Comment by emily-phet** [comment]
Hi folks, I'll be traveling until the 10th starting tomorrow, so I won't be able to do anything with this until after I return.

**Comment by ariel-phet** [comment]
@zepumph please review this issue and see where things are at (for instance I do not see an accessibility category yet), and make a list of the outstanding items so we can work on them.

**Comment by zepumph** [comment]
@ariel-phet  I agree with you, an accessibility section would be nice. I also think that this page would be easy to make pretty with help center like header toggles. It would make the whole page much less unwieldy, and smaller. It wouldn't be so difficult I think. 

In terms of organization, I briefly reviewed and it looked like everything was still organized well in terms of date (newest highest). I also think that most of the links point to the direct site we want, (I don't know exactly what doi.org is, but many point there) but it would also be good to look through for 3rd party links, and for the 'Other Research' papers without links.  

So to overview on my thoughts:

- [ ] Accessibility Research section
- [ ] Revamp the Research Page and make it pretty
- [ ] Overview the organization
  - Make sure things are in chronologic order
  - Make sure that links aren't pointing the 3rd party orgs
- [ ] Add Links in the Other Researchers section to the papers that don't have them.

**Comment by ariel-phet** [comment]
@zepumph all sounds good to me. I would say run with your thoughts on the dev site, and @emily-phet and I can review when you feel you have some good changes. The help center header toggles sound reasonable to me and would make the navigation a bit more seamless/modern.

**Comment by emily-phet** [comment]
Just a note - in case this wasn't clear - the publications under the accessibility section would be the same publications listed on the "research and design" portion of PhET's accessibility section of the website.
Also, I spoke with @amanda-phet yesterday about some of the research page design, and possible icons (similar to the help page icons). Sounds like these icons are still being designed, but concepts exist for all of them.

**Comment by zepumph** [comment]
Sounds great guys. I'll get a draft going and use placer icons, @emily-phet let me know when you have those set.

**Comment by emily-phet** [comment]
@zepumph I gave feedback on the designs, but the person in charge of the icons is @amanda-phet, she can let you know when they're ready to go.

**Comment by amanda-phet** [comment]
What are the top-level categories that we want icons for? I'm trying to parse the comments above, and I think they should be:

* About
* Publications and Presentations

Under "About" is where you'll find the FAQ and Immediate Interests sections, and under "Publications and Presentations" is where you'll find sections for 

    Important features for effective simulation design (predominantly interview data)
    Research on in-class use
    About PhET sims
    Students Perceptions About Learning
    Other Work by PhET Researchers
    Publications on PhET Simulations by Other Researchers

as well as new sections for Research on Accessibility and Outside Research. 

Is that correct?

**Comment by emily-phet** [comment]
@amanda-phet - I apologize.  I realize I was misremembering our conversation about icons. We didn't discuss icons for this page, it was for the accessibility page. Sorry for any confusion!

Yes, I think the top level categories would be "About" and "Publications and Presentations" as you list above. I think it would be more clear, and consistent with common phrasing, to change the sections under "About" to be "FAQ" and "Current Research Topics". (Note: updated text is needed for these)

Under "Publications and Presentations", I think the categories could be worded more concisely (and ordered a bit differently). How about:

- About PhET Sims
- Classroom Research
- Sim Design
- Sim Accessibility
- Other Publications by PhET Researchers
- Other Publications about PhET Sims

The student perceptions category is mostly research not directly related to sims. The papers not related to sims could be listed under "Other Publications by PhET Researchers" and the ones involving sims could be distributed to the remaining categories as appropriate.

**Comment by amanda-phet** [comment]
No worries! Dave designed two icons for this page. Let me know if you would like any other design work done. 
<img width="239" alt="screen shot 2016-11-02 at 11 56 33 am" src="https://cloud.githubusercontent.com/assets/7852916/19940875/af768ec4-a0f3-11e6-902f-51fc9b2326e8.png">

The "About" icon will be used in a few other places on the website as well, but this is the first instance of it so if you have feedback about it we can still make changes to it. "Publications and Presentations" is unique but again, let me know if you would like to make any changes.

**Comment by emily-phet** [comment]
@amanda-phet - these look great!
